django_netjsonconfig/utils.py

```diff
@@ -143,8 +146,45 @@ def get_controller_urls(views_module):
     return urls
 
 
+def get_api_urls(views_module):
+    urls = [
+        url(r'^api/v1/templates/subscribe/$',
+            views_module.notify_template,
+            name='notify_template'),
+        url(r'^api/v1/templates/synchronize/$',
+            views_module.synchronize_template,
+            name='synchronize_template'),
+        url(r'^api/v1/templates/(?P<uuid>[^/]+)/$',
+            views_module.template_detail,
+            name='template_detail'),
+        url(r'^api/v1/templates/$',
+            views_module.list_template,
+            name='list_template')
+    ]
+    return urls
+
+
 def get_random_key():
     """
     generates a device key of 32 characters
     """
     return get_random_string(length=32)
+
+
+def get_remote_template_data(remote_url):
+    """
+    Gets the template data from the
+    remote serialization API
+    """
+    try:
+        response = requests.get(remote_url)
+    except ConnectionError:
+        raise ValidationError({'url': 'Connections to the server with this URL has issues'})
+    if response.status_code == 404:
```

Comment:
you should do something like `if response.status_code != 200:`, only `200` should be good.
In case is not 200 you should include info about the failure in the `ValidationError` message